documentation/modules/appendix_crds.adoc

```diff
@@ -2184,13 +2191,28 @@ It must have the value `tls` for the type `KafkaUserTlsClientAuthentication`.
 |string
 |====
 
+[id='type-KafkaUserTlsNoopClientAuthentication-{context}']
+### `KafkaUserTlsNoopClientAuthentication` schema reference
+
+Used in: xref:type-KafkaUserSpec-{context}[`KafkaUserSpec`]
+
+
+The `type` property is a discriminator that distinguishes use of the `KafkaUserTlsNoopClientAuthentication` type from xref:type-KafkaUserTlsClientAuthentication-{context}[`KafkaUserTlsClientAuthentication`], xref:type-KafkaUserScramSha512ClientAuthentication-{context}[`KafkaUserScramSha512ClientAuthentication`].
```

Comment:
```suggestion
The `type` property is a discriminator that distinguishes use of the `KafkaUserTlsNoopClientAuthentication` type from xref:type-KafkaUserTlsClientAuthentication-{context}[`KafkaUserTlsClientAuthentication`] and xref:type-KafkaUserScramSha512ClientAuthentication-{context}[`KafkaUserScramSha512ClientAuthentication`].
```

Reply:
I will have a look. But I think this is an autogenerated text. So it probably should not be changed as part of this PR. I can try to have a look at it in separate PR.